### ISSUE TITLE
When the primary key is UUID, not ID

### ISSUE THREAD

**Issue by sniper-10** [chelout/laravel-relationship-events#34]
Hello, I have a problem with package,When the primary key is UUID, not ID,This can lead to bugs
    public function detach($ids = null, $touch = true)
    {
        // Get detached ids to pass them to event
        $ids = $ids ?? $this->parent->{$this->getRelationName()}->pluck('id');
         ......
        return $result;
    }

**Comment by chelout** [comment]
Guess, you are right. There should be something like this:
```php
$ids = $ids ?? $this->parent->{$this->getRelationName()}->pluck($this->relatedKey);
```

I'll try to look into this problem soon.

**Comment by chelout** [comment]
Fixed in [1.2.1](https://github.com/chelout/laravel-relationship-events/releases/tag/v1.2.1)

@sniper-10 could you confirm if everything is fine?

**Comment by sniper-10** [comment]
@chelout yes,everything is fine.
